style.css

```diff
@@ -0,0 +1,680 @@
+}
+
+.navmenu li a {
+  text-decoration: none;
+  color: rgba(66, 54, 54, 0.65);
+}
+
+.navmenu li .items .item1 a .about-link {
+  color: red;
+}
+
+li .items .item1 {
+  padding: 0;
+}
+
+#about {
+  color: red;
+}
+
+.school-link {
+  padding: 12px;
+  border: 4px solid red;
+}
+
+#school {
+  color: red;
+}
+
+.navmenu li a:hover {
+  background: #272a31;
+  color: #ec5242;
+}
+
+.ham {
+  cursor: pointer;
+  height: 1.3125rem;
+  margin-left: 9%;
+  align-self: flex-start;
+  margin-top: 2%;
+  margin-bottom: 11%;
+  z-index: 100;
+}
+
+.bar {
+  display: block;
+  width: 45px;
+  height: 7px;
+  background-color: rgba(40, 40, 40, 1);
+  margin: 5px auto;
+}
+
+.ham.active .bar:nth-child(1) {
+  transform: translateY(13px) rotate(45deg);
+  height: 15%;
+  width: 60%;
+  background: red;
+}
+
+.ham.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.ham.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  height: 15%;
+  width: 60%;
+  background: red;
+}
+
+.hamb {
+  display: flex;
+  justify-content: space-between;
+}
+
+#ham {
+  margin-left: 4%;
+}
+
+#close {
+  display: none;
+  margin-left: 4%;
+}
+
+.pages {
+  z-index: 10;
+  height: 99vh;
+  font-size: 30px;
+  font-weight: 600;
+  display: none;
+}
+
+.uon-logo-f {
+  display: none;
+}
+
+.first {
+  padding-top: 10%;
+  padding-left: 4%;
+  padding-right: 3%;
+  background: url(./imgs/Ellipse1.png) no-repeat 30% 40%, url(./imgs/yuyu.png) no-repeat 50% 5%, url(./imgs/twitter.jpg) no-repeat 5% 70%, url(./imgs/Ellipse5.png) no-repeat 70% 5%, url(./imgs/Linkedin-icon.jpg) no-repeat 20% 30%, linear-gradient(60deg, rgb(200, 196, 196), rgb(227, 222, 222), rgb(235, 231, 231));
+  position: relative;
+}
+
+.intro {
+  z-index: 14;
+}
+
+.intro h3 {
+  z-index: 14;
+  font-size: 140%;
+  color: var(--second);
+  line-height: 30px;
+}
+
+.title {
+  z-index: 14;
+  font-size: 400%;
+  font-weight: 700;
+  font-family: 'cocogoose', sans-serif;
+  letter-spacing: 5px;
+  line-height: 60px;
+  color: orangered;
+}
+
+.short-info {
+  z-index: 14;
+  margin-top: 30px;
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 1.5rem;
+  color: rgb(105, 105, 105, 1);
+  background: rgba(240, 240, 240, 0.3);
+  border: 4px solid whitesmoke;
+  padding: 3%;
+}
+
+.date {
+  z-index: 14;
+  margin-top: 30px;
+  font-size: 200%;
+  font-weight: 700;
+  font-family: 'Poppins', sans-serif;
+  color: var(--first);
+  line-height: 40px;
+}
+
+.location {
+  z-index: 4;
+  margin-top: 1%;
+  font-size: 1.2rem;
+  font-weight: 600;
+  color: rgb(105, 105, 105, 1);
+  padding-bottom: 11%;
+}
+
+.overle {
+  width: 100%;
+  height: 100%;
+  background: #cbc7c753;
+  opacity: 0.9;
+  position: absolute;
+  top: 0;
+}
+
+.second {
+  background: url(./imgs/dark.jpg) no-repeat;
+  background-size: cover;
+  color: var(--third);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+
+hr {
+  width: 10%;
+  background: var(--second);
+}
+
+.programs article {
```

Comment:
- [ ] good work 👍🏼 implementing the border as the first reviewer suggest however the middle program items broke when hovering over them, please consider fixing them

![Screenshot_20220528_045943](https://user-images.githubusercontent.com/25227022/170809102-e929b094-1874-4420-8920-7cab19ab6294.png)